src/axelar/interfaces/IAxelarTransceiver.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+import { ITransceiver } from '@wormhole-foundation/native_token_transfer/interfaces/ITransceiver.sol';
+
+interface IAxelarTransceiver is ITransceiver {
+    error InvalidChainId(uint16 chainId);
+
+    event SendTransceiverMessage(uint16 indexed recipientChainId, bytes nttManagerMessage, bytes32 indexed recipientNttManagerAddress, bytes32 indexed refundAddress);
```

Comment:
is there a requirement for this event (does wormhole transceiver emit the same event as well)? If not, maybe add the axelar destination chain and address as well?

Reply:
Wormhole Transceiver does emit this event -- https://github.com/wormhole-foundation/example-native-token-transfers/blob/main/evm/src/Transceiver/WormholeTransceiver/WormholeTransceiver.sol#L240

I think it'd be ideal to follow the same event structure for indexers / explorers.